control/freqplot.py

```diff
@@ -228,12 +233,12 @@ def bode_plot(
 
     Notes
     -----
-    1. Starting with python-control version 0.10, `bode_plot`returns an
-       array of lines instead of magnitude, phase, and frequency. To
-       recover the old behavior, call `bode_plot` with `plot=True`, which
-       will force the legacy values (mag, phase, omega) to be returned
-       (with a warning).  To obtain just the frequency response of a system
-       (or list of systems) without plotting, use the
+    1. Starting with python-control version 0.10, `bode_plot`returns a
```

Comment:
```suggestion
    1. Starting with python-control version 0.10, `bode_plot` returns a
```
This space is important because without it, Sphinx fails to parse the `:class:` expression.

Reply:
After fixing this, the PR is ready to merge.

Reply:
You may want to commit the figures in the documentation that are affected by this PR.